src/src/components/SelectFormPopper/SelectFormPopper.tsx

```diff
@@ -0,0 +1,173 @@
+// @ts-nocheck
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+import ToggleButton from '@material-ui/lab/ToggleButton';
+import {
+  Checkbox,
+  InputBase,
+  Popper,
+  Snackbar,
+  Tooltip,
+} from '@material-ui/core';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import MuiAlert from '@material-ui/lab/Alert';
+import CheckBoxOutlineBlank from '@material-ui/icons/CheckBoxOutlineBlank';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+
+import { Text } from 'components/kit';
+
+import { ISelectOption } from 'types/services/models/explorer/createAppModel';
+
+const SelectFormPopper: React.FC<ISelectFormPopperProps> = ({
+  id,
+  type,
+  open,
+  anchorEl,
+  options,
+  selectedData,
+  onSelect,
+  searchValue,
+  handleSearchInputChange,
+  handleClose,
+  regexError,
+  setRegexError,
+  isRegexSearch,
+  setIsRegexSearch,
+  classes,
+}) => {
+  return (
+    <Popper
+      id={id}
+      open={open}
+      anchorEl={anchorEl}
+      placement='bottom-start'
+      className={
+        type === 'metrics'
+          ? 'Metrics__SelectForm__Popper'
+          : 'SelectForm__Popper'
+      }
+    >
+      <Autocomplete
+        open
+        onClose={handleClose}
+        multiple
+        size='small'
+        disablePortal
+        disableCloseOnSelect
+        options={options}
+        value={selectedData?.options}
+        onChange={onSelect}
+        classes={{
+          popper: classes.popper,
+        }}
+        groupBy={(option) => option.group}
+        getOptionLabel={(option) => option.label}
+        renderTags={() => null}
+        disableClearable={true}
+        ListboxProps={{
+          style: {
+            height: 400,
+          },
+        }}
+        renderInput={(params) => (
+          <div style={{ display: 'flex', alignItems: 'center' }}>
+            <Tooltip title='Select all'>
```

Comment:
Possibly "Select all visible" would be clearer? Since it only applies to the filtered view